src/api.js

```diff
@@ -356,7 +360,23 @@ class API {
     switch (result.event) {
       case 'success': {
         // Connection success
-        this._triggerCallback(CALLBACK_CONNECTION, result.account);
```

Comment:
I think we should not havve to change anything here. It would still take `result.account` from Vezgo Connect, it's just that  `result.account` would be a single account id, or the result object like we discussed. This would keep the implementation details here simple